README.md

```diff
@@ -17,21 +17,93 @@ The following commands can be run within a Cargo package or at the root of a Car
 Build crate with all feature flag combinations:
 
 ```
-cargo build-all-features <CARGO BUILD FLAGS>
+cargo all-features build -- <CARGO BUILD FLAGS>
 ```
 
 Check crate with all feature flag combinations:
 
 ```
-cargo check-all-features <CARGO CHECK FLAGS>
+cargo all-features check -- <CARGO CHECK FLAGS>
 ```
 
 Test crate with all feature flag combinations:
 
 ```
-cargo test-all-features <CARGO TEST FLAGS>
+cargo all-features test -- <CARGO TEST FLAGS>
 ```
 
+<details>
+    <summary markdown="title"><bold>Supported tools</bold></summary>
+
+- First party
+    - [`cargo test`](https://doc.rust-lang.org/cargo/commands/cargo-test.html) cargos integrated testing tool
+    - [`cargo check`](https://doc.rust-lang.org/cargo/commands/cargo-check.html) cargos integrated checking tool
+    - [`cargo build`](https://doc.rust-lang.org/cargo/commands/cargo-build.html) cargos integrated build tool
+    - `cargo bench` [Used by cargos benching feature](https://doc.rust-lang.org/cargo/commands/cargo-bench.html) or crates like [citerion](https://github.com/bheisler/criterion.rs)
+- Additional RustUp components
+    - [`cargo miri test`](https://github.com/rust-lang/miri) for testing using miri -> _rustup component `miri` is needed_
+- Cargo plugins
+    - [`cargo udeps`](https://github.com/est31/cargo-udeps) to analyze for unused dependencies -> _cargo plugin `cargo-udeps` is needed_
+    - [`cargo tarpaulin`](https://github.com/xd009642/tarpaulin) to analyze for unused dependencies -> _cargo plugin `cargo-tarpaulin` is needed_
+    - [`cargo nextest`](https://nexte.st/) the next generation test runner for cargo -> _cargo plugin `cargo-nextest` is needed_
+
+> for more information run `cargo all-features --help`
+</details>
+
+<details>
+    <summary markdown="span">Additional Features</summary>
+
+### Chunking
+
+If certain projects, features might add up and CI jobs can take longer. In order to shrink wall time of your builds you can specify `--chunks` (the total amount of junks to split into _[1..]_) and `--chunk` (the chunk nr of the one executed command _\[1..\<CHUNKS\>\]_) per execution.
+
+I.e. in github you can use a job matrix:
+
+```yaml
+name: CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chunk: [1,2,3,4]
+        chunks: 4
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+     - name: Install cargo-all-features
+        uses: actions-rs/cargo@v1
+        with:
+          command: install cargo-all-features
```

Comment:
I wonder if we should suggest passing the argument that specifies a version to install. That could make it easier to introduce breaking changes in the future 🤔

Reply:
I'm a fan of the idea, but I do not really think many users will manually change the version inside the commands input as this would not be covered by something like dependabot.

One suggestion would be create a gh action for cargo-all-features itself, this could mean a 1:1 mapping between gh-action version and crate version which then could be updated using dependabot.